content/en/real_user_monitoring/session_replay/heatmaps.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Heatmaps
+description: Heatmaps are a type of visualization allowing you to see where users click on your website.
+aliases:
+- /real_user_monitoring/heatmaps
+further_reading:
+- link: '/real_user_monitoring/session_replay/browser/'
+  tag: 'Documentation'
+  text: 'Session Replay for Browsers'
+- link: '/real_user_monitoring/session_replay/mobile/'
+  tag: 'Documentation'
+  text: 'Session Replay for Mobile'
+- link: 'https://www.datadoghq.com/blog/visualize-behavior-datadog-scrollmaps/'
+  tag: 'Blog'
+  text: 'Visualize user interactions with your pages by using Scrollmaps in Datadog Heatmaps'
+---
+
+{{< img src="real_user_monitoring/heatmaps/heatmap_v2.png" alt="An overview of the heatmap functionality." style="width:100%;">}}
+
+A heatmap (or heat map) is a visualization of your user's interactions overlaid on Session Replay data. Real User Monitoring (RUM) has three different types of heatmaps:
+
+- **Click maps:** View user interactions (clicks) to understand how users engage with your page
+- **Top Elements:** View a ranking of up to the top 10 most interacted-with elements on a given page.
+- **Scroll maps:** View how far users scroll down a page, including where the average fold of a page falls. The average fold is the lowest point on a page that a user can see on their device without scrolling.
+
+Use heatmaps to review complex data at a glance, gaining insights around optimizing your user experience.
+
+## Prerequisites
+
+To get started with heatmaps:
+
+1. Verify your SDK version:
+  - For Click maps, you must be on the latest version of the SDK (v4.40.0 or later).
+  - For Scroll maps, you must be on (v4.50.0 or later)
+2. Enable [Session Replay][1].
+3. Set`trackUserInteractions: true` in the SDK initialization to enable action tracking (required for Clickmaps).
+
+## Getting started
+
+Navigate to [**Digital Experience > Real User Monitoring > Session Replay > Heatmaps**][2]. Select your application and view.
+
+On the [Real User Monitoring landing page][3], select your application and view. You can select the type of heatmap you would like to view as well (Top Elements, Click Map, Scroll Map).
+
+{{< img src="real_user_monitoring/heatmaps/rum-heatmaps-getting-started.png" alt="Select an application to view heatmaps for it" style="width:100%;" >}}
+
+This takes you to the [heatmap page][2] for a particular view. You can switch the view being shown with the **View Name** and **Application** selectors at the top. To add more granular filters, like a specific geography for example, you can add a filter from the panel on the left side.
```

Comment:
Can you clarify "This"? I thought I was supposed to be taken to a separate page, but since it's the same link as in ln.40, I realized I was where I'm supposed to be. Also spent some time looking for the Application selector, it's not explicit in the UI.

Reply:
updated - thanks!